src/components/FullPageWebView/FullPageWebView.tsx

```diff
@@ -196,11 +198,26 @@ const FullPageWebView = ( ) => {
     }, [navigation, params.loggedIn, params.title] )
   );
 
+  const fontScalingJS
+  = `
+    if( ${isLargeFontScale} ){
+      document.body.style.fontSize = '40px';
+      let paragraphs = document.querySelectorAll('p');
+      paragraphs.forEach(p => {
+        p.style.fontSize = '50px';
+      })
```

Comment:
Why is modifying the P elements necessary if you're setting a base size for the BODY?

Reply:
Not 100% sure why. Modifying the BODY only seems to affect the headings. My guess is that there might be some css that defines the font size specifically for P tags that overrides the BODY.

Reply:
That sounds like a web page hard-coding a font size... which is kind of out of our control, unless it's our own web page. Where were you seeing this?